src/lib/invoke.js

```diff
@@ -9,6 +11,12 @@ module.exports = (key, service, action, data, isCLI) => {
     'X-Build-Meta-OS': `${os.type().toLowerCase()}@${os.release()}`,
   };
 
+  if (data['x-build-outputs'] !== undefined) {
+    const parsedOutput = querystring.stringify(data['x-build-outputs']);
+    delete data['x-build-outputs'];
+    headers['X-Build-Output'] = parsedOutput;
```

Comment:
Can you add a test here to make sure the header gets set? https://github.com/readmeio/api/blob/master/test/commands/run.test.js

Reply:
@domharrington is this file for the `.run` function?